iatidataquality/organisations.py

```diff
@@ -285,7 +285,7 @@ def agg_detail(agt):
     agg_type = [agg_detail(agt) for agt in all_aggregation_types]
 
     timeliness_score = get_timeliness_score(organisation.frequency, organisation.timelag)
-    max_points = round(33.3 + 66.7 * timeliness_score, 1)
+    max_points = round(33.3 + 66.7 * timeliness_score, 2)
```

Comment:
@Bjwebb, should the `33.3` here be `33.33`, and the `66.7` be `66.67`? Sections 3.3 and 3.4 of the technical paper use the figures `33.33`and `66.67` in the worked examples.

Reply:
I've updated it to use 100.0/3, which is what we use for the actual calculation.